java-spiffe-core/src/main/java/io/spiffe/svid/x509svid/X509SvidValidator.java

```diff
@@ -61,9 +63,16 @@ public static void verifySpiffeId(@NonNull final X509Certificate x509Certificate
                                       @NonNull final Supplier<Set<SpiffeId>> acceptedSpiffeIdsSupplier)
             throws CertificateException {
         val spiffeIdSet = acceptedSpiffeIdsSupplier.get();
+        if (spiffeIdSet.isEmpty()) {
+            String error = "The supplier of accepted SPIFFE IDs supplied an empty set";
+            log.warning(error);
```

Comment:
Why are we logging here? I mean this seems to be a sort of validation method, so I think it makes sense to throw just the exception, and let the caller log it if needed. For instance, I don't see any logging in the verifyChain method (which I think is a good thing). 
Also, why is it logged at warning level instead of error level? It throws an exception, so it seems an error to me.
Anyway, if we really want to add logging here, we should use debug level IMO.

Reply:
I agree, it's not ideal, I added it there as a warning to increase visibility of those validation errors, because I found that when working with Tomcat, the exception is not shown on the logs except if the `-Djavax.net.debug=ssl` is set, but the problem with that is that is too verbose. I think it's important to make the validation error more visible.

Reply:
I see, it makes sense then.